docs/docs/index.md

```diff
@@ -1,8 +1,8 @@
----
-title: Setting Up and Running Marathon
----
+-----------------------
+title: Install Marathon
+-----------------------
```

Comment:
Note: wrapping the title in two lines containing three dashes `---` each is a markup syntax that results in html metadata containing the key/value pairs in between them. If you use more than 3 dashes, the syntax is broken and does **not** result in the wanted metadata. This also breaks html generation at various points for me. Please fix this in all the changed *.md files